src/item_factory.cpp

```diff
@@ -1109,6 +1113,22 @@ void Item_factory::load_wheel( JsonObject &jo, const std::string &src )
     }
 }
 
+void Item_factory::load( islot_fuel &slot, JsonObject &jo, const std::string &src )
+{
+    bool strict = src == "core";
```

Comment:
The current core plugin is `"dda"`. BTW we would need to abandon the repetitive string checks for this.